### ISSUE TITLE
Tags for blog posts?

### ISSUE THREAD

**Issue by mnunes** [Track3/hermit#12]
Hi! Have you ever wondered to add tags to blog posts? I looked for them in your theme, but I couldn't find how to put them.

**Comment by Track3** [comment]
This theme has two kinds of content template, one is regular pages, the other kind is blog posts. Only content under `content\posts` folder will be treated like blog pages, which has tags support.

See https://github.com/Track3/hermit#manage-content

I just took a look at your site, you should rename your `content/post` folder to `content/posts`, or rename theme's `layouts/posts` folder to match.

**Comment by mnunes** [comment]
Thanks. I tried to do it, but if I change `content/post` folder to `content/posts`, I get an error message: 

    Building sites … ERROR 2018/12/14 09:07:27 render of "page" failed: execute of
    template failed: template: posts/single.html:48:5: executing "main" at <
    partialCached "disqu...>: error calling partialCached: "/Users/marcus/Documents/
    Research/marcusnunes.me/themes/hermit/layouts/partials/disqus.html:10:36": execute
    of template failed: template: partials/disqus.html:10:36: executing "partials/
    disqus.html" at <.Site.DisqusShortnam...>: can't evaluate field Site in type
    string
    ERROR 2018/12/14 09:07:27 render of "page" failed: execute of template failed:
    template: posts/single.html:48:5: executing "main" at <partialCached "disqu...>:
    error calling partialCached: "/Users/marcus/Documents/Research/marcusnunes.me/
    themes/hermit/layouts/partials/disqus.html:10:36": execute of template failed:
    template: partials/disqus.html:10:36: executing "partials/disqus.html" at
    <.Site.DisqusShortnam...>: can't evaluate field Site in type string
    ERROR 2018/12/14 09:07:27 render of "page" failed: execute of template failed:
    template: posts/single.html:48:5: executing "main" at <partialCached "disqu...>: 
    error calling partialCached: "/Users/marcus/Documents/Research/marcusnunes.me/
    themes/hermit/layouts/partials/disqus.html:10:36": execute of template failed: 
    template: partials/disqus.html:10:36: executing "partials/disqus.html" at 
    <.Site.DisqusShortnam...>: can't evaluate field Site in type string
    ERROR 2018/12/14 09:07:27 render of "page" failed: execute of template failed: 
    template: posts/single.html:48:5: executing "main" at <partialCached "disqu...>: 
    error calling partialCached: "/Users/marcus/Documents/Research/marcusnunes.me/themes
    /hermit/layouts/partials/disqus.html:10:36": execute of template failed: template: 
    partials/disqus.html:10:36: executing "partials/disqus.html" at 
    <.Site.DisqusShortnam...>: can't evaluate field Site in type string

It is related with `partials/disqus.html`, but I couldn't figure out what is happening.

**Comment by Track3** [comment]
Yes, it's a mistake, and it's fixed in #9, update the theme should work.
